build-support/githooks/pre-commit

```diff
@@ -55,17 +55,18 @@ echo "* Checking for bad shell patterns" && ./build-support/bin/check_shell.sh |
 
 # When travis builds a tag, it does so in a shallow clone without master fetched, which
 # fails in pants changed.
-if git rev-parse --verify "master" &>/dev/null; then
+MERGE_BASE="$(git_merge_base)"
```

Comment:
I think this would fit better at line 36, right after `ADDED_FILES`, because you're declaring in that section the variables we'll use later that involve "what was changed".

Reply:
Done.